src/service/middlewares/article-validator.js

```diff
@@ -11,6 +11,7 @@ module.exports = (req, res, next) => {
   if (!keysExist) {
     res.status(HttpCode.BAD_REQUEST)
       .send(`Bad request`);
+    return;
```

Comment:
```suggestion
    return res.status(HttpCode.BAD_REQUEST)
      .send(`Bad request`);
  
```
так красивей, и так делают чаще в реальных проектах. но можешь не править. чисто up to you